src/tenderly/deploy-verify-zns.ts

```diff
@@ -0,0 +1,50 @@
+import { deployZNS } from "../../test/helpers";
+import * as hre from "hardhat";
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+
+
+export const deployVerifyZNS = async ({
+  governor,
+  governorAddresses = [ governor.address ],
+  adminAddresses = [ governor.address ],
+} : {
+  governor : SignerWithAddress;
+  governorAddresses ?: Array<string>;
+  adminAddresses ?: Array<string>;
+}) => {
+  const zns = await deployZNS({
+    deployer: governor,
+    governorAddresses,
+    adminAddresses,
+  });
+
+  // TODO: find a way to get these names from the object
+  const names = [
+    "ZNSAccessController",
+    "ZNSRegistry",
+    "ZNSDomainToken",
+    "ZeroTokenMock",
+    "ZNSAddressResolver",
+    "ZNSPriceOracle",
+    "ZNSTreasury",
+    "ZNSEthRegistrar",
+  ];
+
+  // Verify and add source code to Tenderly
+  // for easy debugging
+  await Object.values(zns).reduce(
+    async (acc, contract, idx) => {
```

Comment:
Why is `acc` included? It doesn't seem to get referenced

Reply:
because you need to specify other params that are after it.